src/services/class.service.ts

```diff
@@ -26,6 +26,8 @@ export class ClassService {
   private _classVoiceChans: Map<string, ClassVoiceChan> = new Map();
   private _AUDIO_CAT: Maybe<CategoryChannel> = null;
 
+  private _LEVENSHTEIN = require('js-levenshtein');
```

Comment:
prefer `import` instead

Reply:
`import * as levenstein from 'js-levenstein'`

Reply:
I too like import over require, however import does not seem to work.  The only example of an import I have seen is `import levenstein from 'https://unpkg.com/js-levenshtein-esm/js-levenshtein.js';` and that does not seem to work.

Reply:
In that case, just use
```
import levenshtein from 'js-levenshtein';
```

Reply:
Then you would have
```
return classes
      .sort((a: string, b: string) => levenshtein(className, a) - levenshtein(className, b))
      .slice(0, 5);
```

Reply:
might have to do 

```
import {levenshtein} from '...'
```
if there are multiple exports,